### PR TITLE
fix(listing): Added Server Side Validation for Classifications #149

### PR DESCRIPTION
**Pull request by mannyrivera2010**

### CODE REVIEW

**Comment by mannyrivera2010** [comment]
For Issue #149 
Please Review:
@rkenyon1969 
@clarkjefcoat

File: ozpcenter/access_control.py [hunk 1]
```diff
@@ -17,6 +24,151 @@
...
+        short_name_lookup[token.short_name.upper()] = token
+
+    markings = input_marking.split(delimiter)
+
+    output_tokens = []
+    for marking in markings:
+        current_token = None
+
+        if marking.upper() in long_name_lookup:
+            current_token = long_name_lookup[marking]
```
Comment (clarkjefcoat) [inline ozpcenter/access_control.py]: Don't you need
    current_token = long_name_lookup[marking.upper()]
?

File: ozpcenter/access_control.py [hunk 2]
```diff
@@ -17,6 +24,151 @@
...
+        else:
+            if marking.upper() in short_name_lookup:
+                current_token = short_name_lookup[marking]
```
Comment (clarkjefcoat) [inline ozpcenter/access_control.py]: Same comment as line 143

File: ozpcenter/access_control.py (hunk 1 above)
Comment (mannyrivera2010) [inline ozpcenter/access_control.py]: Yes, that is correct, I pushed the change

**Comment by clarkjefcoat** [comment]
:+1:

**Comment by rkenyon1969** [comment]
:+1:

**Pull request merged**